go.mod

```
@@ -2,13 +2,15 @@ module github.com/hashicorp/terraform-provider-nomad
 
 go 1.21
 
-// Set to the version used by Nomad
-// https://github.com/hashicorp/nomad/blob/v1.6.0-rc.1/go.mod#L74
-replace github.com/hashicorp/hcl/v2 => github.com/hashicorp/hcl/v2 v2.9.2-0.20220525143345-ab3cae0737bc
+replace (
+	// Set to the version used by Nomad
+	// https://github.com/hashicorp/nomad/blob/v1.6.0-rc.1/go.mod#L74
+	github.com/hashicorp/hcl/v2 => github.com/hashicorp/hcl/v2 v2.9.2-0.20220525143345-ab3cae0737bc
 
-// Pin Nomad to a commit where jobspec and jobspec2 are lincesed under MPL-2.0.
-// Remove when updating to Nomad 1.7.3+
-replace github.com/hashicorp/nomad => github.com/hashicorp/nomad v0.0.0-20240129172537-56ea3f7cdafd
+    // Fix error tidying due to Nomad downstream dependencies and the recent
+    // migration of the metrics library.
```

bit of inconsistent spacing here

```suggestion
	// Fix error tidying due to Nomad downstream dependencies and the recent
	// migration of the metrics library.
```

I think that gets em to all be tabs?

---

Fixed in: https://github.com/hashicorp/terraform-provider-nomad/pull/462/commits/67d75b858ab972c08d1b6d81646a258eb30a2986